### PR TITLE
Fixes for RFAI filings

### PR DESCRIPTION
**Pull request by patphongs**
## Summary (required)

- Addresses #1760 
Ensuring that the following RFAIs show up in the correct tables.
### Missing from candidate pages
RQ-5s - Request for Additional Information (RFAI) referencing the Statement of Candidacy. Test URL: http://localhost:8000/data/candidate/H8AR04122/?tab=about-candidate
### Missing from committee pages
RQ-1 - RFAI referencing the Statement of Organization. Test URL: http://localhost:8000/data/committee/C00637983/?tab=filings
RQ-2 - RFAI referencing Report of Receipts and Expenditures. Test URL: http://localhost:8000/data/committee/C00637983/?tab=filings
RQ-9 - Multicandidate status. Test URL: http://localhost:8000/data/committee/C00659672/?tab=filings

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Committee filing page
- Candidate about page

### RFAIs of RQ 2 Only
screen shot 2018-03-02 at 10 06 17 am

### RFAIs of RQ 1 Only
screen shot 2018-03-02 at 10 06 27 am

### RFAIs of RQ 9 Only
screen shot 2018-03-02 at 10 05 58 am

### RFAIs of RQ 5 Only
screen shot 2018-03-04 at 10 36 26 pm

### CODE REVIEW

**Comment by lbeaufort** [comment]
@patphongs disregard - this trips me up every time. Once I tested out I realized that specifying the request type would return only those RQ-X's and not the reports as well. Looks good so far!

**Comment by lbeaufort** [comment]
Great work, @patphongs! Your PR description and local testing links were very helpful.

File: fec/fec/static/js/pages/candidate-single.js [hunk 1]
```diff
@@ -207,7 +207,10 @@ function initOtherDocumentsTable() {
     path: path,
     query: {
       candidate_id: candidateId,
-      form_type: 'F99'
+      form_type: ['F99','RFAI'],
+      //exclude all request types except for RQ-5: RFAI referencing Statement of Candidacy
+      request_type: ['-1','-2','-3','-4','-6','-7','-8','-9'],
```
[the remark this answers is not in the document]
Comment (patphongs) [reply fec/fec/static/js/pages/candidate-single.js]: Thanks @lbeaufort! Sounds like you figured it out why I had to do it this way. Let me know if you have any further questions.

**Comment by llienfec** [comment]
Great work on this @patphongs and thanks for the thoughtful comments @lbeaufort.

**Pull request merged**